<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ib_changePassword</class>
 <widget class="QWidget" name="ib_changePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>336</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="changepassword_bar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#changepassword_bar {
    border-style:solid;
	border-width:1px;
	border-color:rgb(187, 187, 187);
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(242, 242, 242), stop:1 rgb(221, 221, 221));
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_icon">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/Images/logo.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_changepassword">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>修改密码</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_bar">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="combination" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#combination {
    border-style:solid;
	border-width:1px;
	border-color:rgb(187, 187, 187);
    background-color: white;
    border-top: none;
}</string>
     </property>
     <widget class="QPushButton" name="btn_confirm">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clo">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>271</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lb_old">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>旧密码</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_old">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_new">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>新密码</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_new">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_confirm">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_confirm">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
